build-farm/platform-specific-configurations/linux.sh

```diff
@@ -78,7 +78,18 @@ fi
 
 if [ "${ARCHITECTURE}" == "arm" ]
 then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-jobs=4 --with-memory-size=2000"
+  if lscpu | grep aarch64; then
```

Comment:
Should we `[` bracket this if statement to match style?

Reply:
No - this is an external commend which we are testing the return code is (as `[` is when doing logical operations) and is not doing a `test` operaion, so that would not have the desired effect.